packages/unenv-preset/tests/index.test.ts

```diff
@@ -0,0 +1,49 @@
+import path from "node:path";
+import { platform } from "node:process";
+import { fileURLToPath } from "node:url";
+import { fetch } from "undici";
+import { afterAll, beforeAll, describe, expect, test } from "vitest";
+import { runWranglerDev } from "../../../fixtures/shared/src/run-wrangler-long-lived";
+import { TESTS } from "./worker/index";
+
+const pkgDir = path.resolve(fileURLToPath(import.meta.url), "../..");
+const localWorkerdPath = path.join(pkgDir, "node_modules/.bin/workerd");
+// Skip tests on windows where MINIFLARE_WORKERD_PATH is not supported
+const isWindows = platform === "win32";
+
+describe("@cloudflare/unenv-preset", () => {
+	let wrangler: Awaited<ReturnType<typeof runWranglerDev>>;
+
+	beforeAll(async () => {
+		if (isWindows) {
+			return;
+		}
```

Comment:
Can you not just use `describe.skipIf(isWindows)` rather than adding this in every hook and test?

Reply:
Actually, is it not possible to make these tests work on Windows?
You say that MINIFLARE_WORKERD_PATH is not supported on Windows. Is that strictly accurate or is it that you just need to specify a different path for Windows (e.g. `"node_modules/.bin/workerd.cmd"` or `"node_modules/.bin/workerd.ps1"`)?

Reply:
I think that comment in Miniflare is not the same thing. What they are saying there (I believe) is that their test uses a shebang binary to trigger workerd, which would not work on Windows. But when you install workerd it should provide a Windows compatible binary, which usually ends in .cmd or .ps1, and that ought to work in this scenario.

I am not convinced that testing on Max+Ubuntu is adequate to ensure Windows compatibility 😆

Reply:
> I am not convinced that testing on Max+Ubuntu is adequate to ensure Windows compatibility 😆

What I mean is that I don't really care about the dev version working on Windows because I don't think any use of dev on Windows. Wrangler is tested on windows which should be what matters.

But I'll see if there is something I can do without spending too much time.